addons/pinniped/config-controller/README.md

```diff
@@ -15,4 +15,18 @@ $ ./hack/test.sh
 To run static checks (e.g., `go fmt`, `go vet`, `golangci-lint`) and the tests:
 ```sh
 $ ./hack/check.sh
-```
\ No newline at end of file
+```
+
+To generate the default pinniped addon secret:
+
+```sh 
+# manually via ytt
+ytt -f ./hack/ytt/default-package-secret.yaml -f ./hack/ytt/values.yaml -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere
+
+# using the generate script
+./hack/generate-package-secret.sh -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere 
```

Comment:
should this `iaas` be `infrastructure_provider` since that is what the YTT data value is? is it worth just generally documenting that the script takes YTT args so that this doesn't get out of date?

Reply:
ill comment something like this, prob leave as an example but imply "its just an example".